packages/app/src/Pages/Login.tsx

```diff
@@ -279,12 +279,20 @@ export default function LoginPage() {
           <div className="flex">
             <input
               dir="auto"
-              type="text"
+              type={key?.startsWith("nsec") ? "password" : "text"}
```

Comment:
We also allow logging in with NIP-06 menemic, that should be masked too, but also makes it hard to enter if you can just typing the words.

It might be difficult to implement this properly with a single input.

Reply:
To properly implement privacy for nsec and mnemonics, I did the following things:
1. Display an 'eye' icon next to the input field to reveal or hide the input, when the user clicks it
2. When the user enters a nsec or mnemonic, show a suggestion to hide the input using the 'eye' icon (if it is not hidden already)